edc/README.md

```diff
@@ -0,0 +1,8 @@
+# EDC Connector Image
```

Comment:
This should mention that it is not part of SDP. Please coordinate with @stackfab or @stefanigel on wording

Reply:
I've labelled it as a "technology preview" component